practice/react-practice-one/src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,71 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import components */
+import LogoIcon from '../Icons/LogoIcon';
+import HomeIcon from '../Icons/HomeIcon';
+import DiscountIcon from '../Icons/DiscountIcon';
+import DashboardIcon from '../Icons/DashboardIcon';
+import MessageIcon from '../Icons/MessageIcon';
+import NotificationIcon from '../Icons/NotificationIcon';
+import SettingIcon from '../Icons/SettingIcon';
+import LogoutIcon from '../Icons/LogoutIcon';
+
+/* Import CSS */
+import './sideBar.css';
+
+const SideBar = ({ onClick, activeItem, setActiveItem }) => {
+  const menuItems = [
+    { name: 'home', icon: HomeIcon },
+    { name: 'dashboard', icon: DashboardIcon },
+    { name: 'discount', icon: DiscountIcon },
+    { name: 'message', icon: MessageIcon },
+    { name: 'notification', icon: NotificationIcon },
+    { name: 'setting', icon: SettingIcon },
+  ];
+
+  const handleClick = (item) => {
+    setActiveItem(item);
+
+    if (onClick) {
+      onClick(item);
+    }
+  };
+
+  return (
+    <aside className="sidebar-overlay">
+      <div className="sidebar-header">
+        <a href="javascript:void(0)" className="sidebar-logo">
```

Comment:
@JennyDoJD pls help remove and double check before requesting review

Reply:
I done